documentation/modules/migrating-virtual-machines-cli.adoc

```diff
@@ -148,10 +148,10 @@ spec:
   provider:
     source:
       name: <source_provider>
-      namespace: {namespace}
+      namespace: <namespace>
     destination:
       name: <destination_cluster>
```

Comment:
```suggestion
      name: <destination_provider>
```
(we use both `destination providers` and `target providers` in our documentation, we can use one of these here)